ui/static/ui/js/lore_data_table.jsx

```diff
@@ -0,0 +1,197 @@
+          sortInfo={this.state.sortInfo}
+          onSortChange={this.onSortChange}
+          dataSourceCount={count}
+          page={this.state.page}
+          onPageChange={this.onPageChange}
+          pageSize={this.state.pageSize}
+          onPageSizeChange={this.onPageSizeChange}
+          onColumnResize={this.onColumnResize}
+          onFilter={this.handleFilter}
+          liveFilter={true}
+          />;
+      },
+      handleFilter: function(column, value, allFilterValues) {
+        this.setState({filterValues: allFilterValues});
+      },
+      onPageChange: function(page) {
+        this.setState({page: page});
+      },
+      onPageSizeChange: function(pageSize) {
+        this.setState({pageSize: pageSize});
+      },
+      onSortChange: function(info) {
+        this.setState({sortInfo: info});
+      },
+      onColumnResize: function(firstCol, firstSize) {
+        var columns = _.map(this.state.columns, function(column) {
+          if (firstCol.name === column.name) {
+            var copy = $.extend({}, column);
+            copy.width = firstSize;
+            return copy;
+          }
+          return column;
+        });
+        this.setState({columns: columns});
+      }
+    });
+
+    var closeLearningResourcePanel = function() {
+      $('.cd-panel').removeClass('is-visible');
+    };
+
+    var loader = function(repoSlug, data, container) {
+      var columns = [
+        {
+          name: "lid",
+          title: "Resource id",
+          type: "number",
+          render: function(lid) {
+            var onclick = function(e) {
+              e.preventDefault();
+              // Unmount and remount the component to ensure that its state
+              // is always up to date with the rest of the app.
+              var resourcePanelDiv = $("#tab-1")[0];
+              React.unmountComponentAtNode(resourcePanelDiv);
+              React.render(<LearningResources.LearningResourcePanel
+                repoSlug={repoSlug}
+                learningResourceId={lid}
+                refreshFromAPI={function() {}}
+                closeLearningResourcePanel={closeLearningResourcePanel}
+                />, resourcePanelDiv);
+              var staticAssetDiv = $("#tab-3")[0];
+              React.unmountComponentAtNode(staticAssetDiv);
+              $('.cd-panel').addClass('is-visible');
+              React.unmountComponentAtNode(staticAssetDiv);
+              React.render(<StaticAssets.StaticAssetsPanel
+                repoSlug={repoSlug} learningResourceId={lid}
+                key={[repoSlug, lid]}
+                />, staticAssetDiv);
+            };
+            return <a target="_blank" onClick={onclick}>{lid}</a>;
+          }
+        },
+        {
+          name: "title",
+          title: "Title",
+        },
+        {
+          name: "description",
+          title: "Description"
+        },
+        {
+          name: "description_path",
+          title: "Description Path",
```

Comment:
Can we label this "Path" or "Learning Resource Path" if you don't think that's clear enough?

"Description Path" doesn't make sense to me.